contracts/access/IAccessControl.sol

```diff
@@ -30,8 +30,8 @@ interface IAccessControl {
     /**
      * @dev Emitted when `account` is granted `role`.
      *
-     * `sender` is the account that originated the contract call. This account bears the admin role (for the granted
-     * role), expect in cases where the role was granted using the internal {AccessControl-_grantRole}.
+     * `sender` is the account that originated the contract call. This account bears the admin role (for the granted role).
```

Comment:
that line is 124 chars, should not pass linting.

Reply:
Interesting, I ran `npm lint:fix` before pushing.

Reply:
could be another issue to look into...